sources/packages/backend/apps/api/src/services/student-scholastic-standings/student-scholastic-standings.service.ts

```diff
@@ -513,25 +514,45 @@ export class StudentScholasticStandingsService extends RecordDataModelService<St
   }
 
   /**
-   * Get scholastic standing summary details.
+   * Get scholastic standing unsuccessful completion weeks summary details.
    * @param studentId student id to retrieve the scholastic standing summary details.
-   * @return student scholastic standing summary details.
+   * @return student scholastic standing unsuccessful completion weeks summary details.
    */
   async getScholasticStandingSummary(
     studentId: number,
-  ): Promise<ScholasticStandingSummary> {
+  ): Promise<ScholasticStandingSummaryDetails> {
     const studentScholasticStandingSummary = this.repo
       .createQueryBuilder("studentScholasticStanding")
       .select(
         "SUM(studentScholasticStanding.unsuccessfulWeeks)::int",
         "totalUnsuccessfulWeeks",
       )
+      .addSelect("offering.offeringIntensity", "offeringIntensity")
       .innerJoin("studentScholasticStanding.application", "application")
       .innerJoin("application.student", "student")
+      .innerJoin("application.currentAssessment", "assessment")
+      .innerJoin("assessment.offering", "offering")
       .where("student.id = :studentId", {
         studentId,
       })
-      .groupBy("student.id");
-    return studentScholasticStandingSummary.getRawOne<ScholasticStandingSummary>();
+      .groupBy("student.id, offering.offeringIntensity");
```

Comment:
When we group by multiple columns, we use addGroupBy. 

But more importantly in this case, group by offeringIntensity alone is sufficient.